tests/integration/namespaced-integration-test.sh

```diff
@@ -3,16 +3,69 @@ set -euo pipefail
 
 # This script is expected to be called from the root folder of Connaisseur
 
+echo 'Creating test namespaces'
+kubectl create namespace ignoredns
+kubectl label ns ignoredns securesystemsengineering.connaisseur/webhook=ignore
+kubectl create namespace validatedns
+kubectl label ns validatedns securesystemsengineering.connaisseur/webhook=validate
+
 echo 'Preparing Connaisseur config...'
 yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' helm/values.yaml tests/integration/namespaced-update.yaml
 echo 'Config set'
 
+echo '### Testing "ignore" label ###'
+
 echo 'Installing Connaisseur...'
 make install || { echo 'Failed to install Connaisseur'; exit 1; }
 echo 'Successfully installed Connaisseur'
 
 echo 'Testing unsigned image in protected namespace...'
-kubectl run pod --namespace verifyns --image=securesystemsengineering/testimage:unsigned >output.log 2>&1 || true
+kubectl run pod --image=securesystemsengineering/testimage:unsigned >output.log 2>&1 || true
+
+if [[ ! "$(cat output.log)" =~ 'Unable to find signed digest for image docker.io/securesystemsengineering/testimage:unsigned.' ]]; then
+  echo 'Failed to deny unsigned image or failed with unexpected error. Output:'
+  cat output.log
+  exit 1
+else
+  echo 'Successfully denied usage of unsigned image'
+fi
+
+echo 'Testing signed image in unlabelled namespace...'
```

Comment:
I might be wrong trying to wrap my head around what needs to be tested here, but doesn't it make sense to test whether an unsigned image would be denied in an unlabelled namespace here?